packages/next/build/index.ts

```diff
@@ -122,7 +122,12 @@ export default async function build(dir: string, conf = null): Promise<void> {
 
   let backgroundWork: (Promise<any> | undefined)[] = []
   backgroundWork.push(
-    telemetry.record(eventVersion({ cliCommand: 'build' })),
+    telemetry.record(
+      eventVersion({
+        cliCommand: 'build',
+        isSrcDir: pagesDir.startsWith('src'),
```

Comment:
I don't think this works, the `pagesDir` is joined with `dir` so it wouldn't match `startsWith`. AFAIK we'd need to do a `path.relative(dir, pagesDir)` here. Probably doesn't hurt to test this.